regression/scripts/ats2-env.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#------------------------------------------------------------------------------#
+# ATS-2 Environment setups
+#------------------------------------------------------------------------------#
+
+case $ddir in
```

Comment:
I didn't know that function definitions could be embedded in case statements - that's pretty neat.